tensorflow/python/ops/math_ops.py

```diff
@@ -354,10 +353,6 @@ def multiply(x, y, name=None):  # pylint: disable=missing-docstring
     A 'Tensor'. Has the same type as 'x'
   """
   
-  return gen_math_ops.mul(x, y, name)
```

Comment:
It looks like this PR is completely removing the implementation of the function

Reply:
When you say remove, do you mean lines 338 and 341?

Reply:
Are you referring to my removal of the lines 338 and 341? Should I add those lines back into the function?

Reply:
My apologies for the mistake!! I added the lines back